index.html

```diff
@@ -0,0 +1,864 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Design responsive</title>
+    <link rel="stylesheet" type="text/css" href="css/reset.css">
+    <link rel="stylesheet" href="./css/styles.css"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <header>
+        <nav class="nav-bar">
+            <div class="nav-logo">
+                <img alt="logo-1023px" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg">
+            </div>
+            <div class="nav-bar-content">
+                <div class="topNav">
+                    <ul>
+                        <li><a href="#">News</a></li>
+                        <li><a href="#">Events</a></li>
+                        <li><a href="#">Business</a></li>
+                        <li><a href="#">AMAs</a></li>
+                        <li><a href="#">Spaces</a></li>
+                        <li><a href="#">Terms & Conditions</a></li>
+                    </ul>
+                    <ul class="nav-bar-end">
+                        <li><a href="#">About</a></li>
+                        <li><a href="#">Team</a></li>
+                        <li><a href="#">Advertise</a></li>
+                        <li><a href="#">Jobs</a></li>
+                        <li><a href="#">Contact</a></li>
+                    </ul>
+                </div>
+                <div class="bottomNav">
+                    <ul>
+                        <li><a href="#">LATEST</a></li>
+                        <li><a href="#">HARD FORK</a></li>
+                        <li><a href="#">PLUGGED</a></li>
+                        <li><a href="#">FUNDAMENTALS</a></li>
+                        <li><a href="#">WORK 2030</a></li>
+                    </ul>
+                        <button>
+                            <i class="search-icon"></i>
+                        </button>
+                        <button class="humbg">
+                            <i class="fa fa-bars" aria-hidden="true"></i>
+                        </button>
+                </div>
+            </div>
+            <a href="#">
+                <div class="nav-message nav-message-announcement" style="pointer-events:none"> 
```

Comment:
Remove inline style

Reply:
- [ ] You still didn't fix this inline styling, please remove it.